packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-glossary.tsx

```diff
@@ -112,6 +120,61 @@ const ContractTypeGlossary = ({ category }: { category: string }) => {
                     },
                 ];
                 break;
+            case 'multiplier':
+                content = [
+                    { type: 'heading', text: localize('Stop out') },
+                    {
+                        type: 'paragraph',
+                        text: localize(
+                            'Your trade will be closed automatically at the nearest available asset price when your loss reaches a certain percentage of your stake, but your loss never exceeds your stake. This percentage depends on the chosen underlying asset and the Multiplier.'
+                        ),
+                    },
+                    { type: 'heading', text: localize('Take profit') },
+                    {
+                        type: 'paragraph',
+                        text: localize(
```

Comment:
@kate-deriv  is it possible to use <Localize here ? :)

Reply:
Of course, will do

Reply:
@kate-deriv the check for the `string` type was only necessary because we wanted to check whether or not `text` is `array` when `type` is `list`. This check can be replaced with `Array.isArray(text)`. And `text.substring(0, 10)` in keys can be replaced with numerical keys which can be added to <Localize components directly. :)

Reply:
Will refactor + add sanity of glossary for Vanillas and Accumulators to QA-checklist